src/init.cpp

```diff
@@ -994,6 +995,13 @@ bool AppInitParameterInteraction()
         }
     }
 
+    // Basic filters are the only supported filters. The basic filters index must be enabled
+    // to serve compact filters
+    if (gArgs.GetBoolArg("-peerblockfilters", DEFAULT_PEERCFILTERS) &&
```

Comment:
Per the change to `-peerblockfilters`, consider renaming `DEFAULT_PEERCFILTERS` to `DEFAULT_PEERBLOCKFILTERS` as well.

```cpp
src/init.cpp:449:    gArgs.AddArg("-peerblockfilters", strprintf("Serve compact block filters to peers per BIP 157 (default: %u)", DEFAULT_PEERCFILTERS), ArgsManager::ALLOW_ANY, OptionsCategory::CONNECTION);
src/init.cpp:1000:    if (gArgs.GetBoolArg("-peerblockfilters", DEFAULT_PEERCFILTERS) &&
src/net_processing.cpp:1993:         gArgs.GetBoolArg("-peerblockfilters", DEFAULT_PEERCFILTERS));
src/net_processing.h:24:static const bool DEFAULT_PEERCFILTERS = false;
```

Reply:
done